### ISSUE TITLE
Snap Tap support for Razer Huntsman Pro (Mini) V3

### ISSUE THREAD

**Issue by darix** [openrazer/razer-drivers#2283]
Details:

https://www.youtube.com/watch?v=Feny5bs2JCg

**Comment by Kyr4l** [comment]
is this feature possible to implement on "unsupported keyboards" via a software workaround ?

**Comment by z3ntu** [comment]
If something there is enabled in the keyboard firmware via Synapse then we can probably add a toggle. If it's just a software concept, then it's out of scope for OpenRazer and a generic tool should be able to do the same.

If you can grab packet captures from Razer Synapse of doing this, I can take a look.

But also seems from the video that this is more a software thing of how the analog key values are interpreted - #1868

**Comment by darix** [comment]
but it is interpreted by the firmware not the driver.

**Comment by ByteDrummer** [comment]
Yeah, at least for the Huntsman V3 Pro line, it's built into the firmware and Synapse doesn't need to be running. They have since rolled out the feature for other keyboards, and it's probably done software side, since you need Synapse to be running.

**Comment by z3ntu** [comment]
Then we really need USB packet captures of this to be able to implement it.

**Comment by z3ntu** [comment]
Somebody has shared protocol details for the snap tap feature, for those who are interested: #2754
